templates/premium.php

```diff
@@ -361,9 +363,9 @@ function statAllocateCostDisplay() {
                 <br/>
 
                 <b>Your Forbidden Seal</b><br/>
-                <?php if(isset($player->forbidden_seal['level'])): ?>
-                    <?= System::$forbidden_seals[$player->forbidden_seal['level']] ?><br/>
-                    <?= $system->time_remaining($player->forbidden_seal['time'] - time()) ?>
+                <?php if($player->forbidden_seal_loaded): ?>
```

Comment:
I wonder if there's ever a case we really don't want to load the forbidden seal (it doesn't trigger an additional DB query) - maybe we should always initialize it so our checks are more like `if($player->forbidden_seal != null)`